libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/navigation/DoubleCounter.kt

```diff
@@ -1,6 +1,6 @@
 package com.mapbox.services.android.navigation.v5.internal.navigation
 
-internal class DoubleCounter(
-    name: String,
-    value: Double
-) : Counter<Double>(name, value)
+internal data class DoubleCounter(
+    private val name: String,
```

Comment:
`private` ?

Reply:
Yes, we no need to access this fields from somewhere. This class is wrapper for one of the event field. And later it just transform to JSON